code/modules/antagonists/heretic/soultrapped_heretic.dm

```diff
@@ -0,0 +1,14 @@
+///a heretic that got soultrapped by cultists. does nothing, other than signify they suck
+/datum/antagonist/soultrapped_heretic
+	name = "\improper Soultrapped Heretic"
+	roundend_category = "Heretics"
+	antagpanel_category = "Heretic"
+	job_rank = ROLE_HERETIC
+	antag_moodlet = /datum/mood_event/fallen_changeling
```

Comment:
You got caught! You got busted! Carlarc eating Armhulen code!
(Nah just joshing, make sure to switch this type to `/datum/mood_event/soultrapped_heretic`